etc/profile-a-l/cointop.profile

```diff
@@ -0,0 +1,50 @@
+# Firejail profile for cointop
+# This file is overwritten after every install/update
+# Persistent local customizations
+include cointop.local
+# Persistent global definitions
+include globals.local
+
+noblacklist ${HOME}/.config/cointop
+
+include disable-common.inc
+include disable-devel.inc
+include disable-exec.inc
+include disable-interpreters.inc
+include disable-proc.inc
+include disable-programs.inc
+include disable-shell.inc
+include disable-write-mnt.inc
```

Comment:
```suggestion
```

Reply:
Oh; that one _does not_ do what I thought it did.  Good catch.

Reply:
https://github.com/netblue30/firejail/blob/afed1f5aac9692382ac02f11ffd78ba5ee7d369c/etc/templates/profile.template#L114